src/prometheus_alert_rules/nvme.rule

```diff
@@ -0,0 +1,32 @@
+---
+groups:
+  - name: nvme
+    rules:
+      - record: available_file_size_percentage
+        expr: |
+          100 *  (node_filesystem_avail_bytes / node_filesystem_size_bytes)
+      - alert: FileSystemPercentUsedWarn
+        expr: |
+          avg_over_time(available_file_size_percentage[5m]) < 20
+        for: 5m
+        labels:
+          severity: warning
+        annotations:
+          summary: Available disk on {{ $labels.mountpoint }} is too low
```

Comment:
I think that we should stick with same format for summary and description, such is:
```yaml
summary: Available disk on {{ $labels.mountpoint }} is too low (instance {{ $labels.instance }})
description: >-
  Available disk percentage on mountpoint({{ $labels.mountpoint }}) {{ $value }} is < 20%
    VALUE = {{ $value }}
    LABELS = {{ $labels }}
```